<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSetUART</class>
 <widget class="QWidget" name="FormSetUART">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="uartNumLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="uartNumComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="baudRateLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRateComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>115200</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="dataBitsLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBitsComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>8</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="stopBitsLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>停止位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitsComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="parityLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校验位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parityComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ODD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EVEN</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="openUARTButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QLabel" name="uartStsLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/prefix1/image/close.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
